zerver/lib/push_notifications.py

```diff
@@ -401,21 +407,21 @@ def send_android_push_notification_to_user(
     )
 
 
-def parse_gcm_options(options: Dict[str, Any], data: Dict[str, Any]) -> str:
+def parse_fcm_options(options: Dict[str, Any], data: Dict[str, Any]) -> str:
```

Comment:
These renames fixing all the "GCM" references to "FCM" would be good as a prep commit. It's been "FCM" for a years now — that was a previous migration — so that's past due anyway.

Reply:
There's still a whole bunch of gcm in function and var names etc. left that might be easier to do as a follow-up after the API migration - but I extracted a bunch of the renaming that was adding noise to the main commit for now.

Reply:
Thanks! That plan SGTM — we've tolerated the outdated names for 5 years now, so no rush to pursue the rest of them synchronously for this migration.